walkthroughs/ecs-service-connectivity/yelb/deployments/platformdeployment/AWS/ECS/yelb-cloudformation-ECS-AppMesh-deployment.yaml

```diff
@@ -0,0 +1,631 @@
+                      LogDriver: awslogs
+                      Options:
+                          awslogs-group: !Ref AWS::StackName
+                          awslogs-region: !Ref AWS::Region
+                          awslogs-stream-prefix: yelb-ui
+                - Name: envoy
+                  Image: !Ref EnvoyImage
+                  Essential: true
+                  User: '1337'
+                  DependsOn:
+                      - ContainerName: xray
+                        Condition: START
+                  Ulimits:
+                      - Name: nofile
+                        HardLimit: 15000
+                        SoftLimit: 15000
+                  HealthCheck:
+                      Command:
+                          - CMD-SHELL
+                          - curl -s http://localhost:9901/server_info | grep state | grep -q LIVE
+                      Interval: 5
+                      Timeout: 10
+                      Retries: 10
+                  LogConfiguration:
+                      LogDriver: awslogs
+                      Options:
+                          awslogs-group: !Ref AWS::StackName
+                          awslogs-region: !Ref AWS::Region
+                          awslogs-stream-prefix: yelb-ui
+                  Environment:
+                      - Name: ENVOY_LOG_LEVEL
+                        Value: debug
+                      - Name: ENABLE_ENVOY_XRAY_TRACING
+                        Value: '1'
+                      - Name: ENABLE_ENVOY_STATS_TAGS
+                        Value: '1'
+                      - Name: ENABLE_ENVOY_DOG_STATSD
+                        Value: '1'
+                      - Name: APPMESH_VIRTUAL_NODE_NAME
+                        Value:
+                            Fn::Join:
+                                - ''
+                                -
+                                    - mesh/
+                                    - !Ref Mesh
+                                    - /virtualNode/
+                                    - 'yelb-ui'
+                                    
+                
+
+
+    CloudWatchLogsGroup:
+        Type: AWS::Logs::LogGroup
+        Properties: 
+            LogGroupName: !Ref AWS::StackName
+            RetentionInDays: 365  
+   
+    YelbECSTaskIamRole:
+        Type: AWS::IAM::Role
+        Properties:
+            Path: /
+            AssumeRolePolicyDocument: |
+                {
+                    "Statement": [{
+                        "Effect": "Allow",
+                        "Principal": { "Service": [ "ecs-tasks.amazonaws.com" ]},
+                        "Action": [ "sts:AssumeRole" ]
+                    }]
+                }
+            ManagedPolicyArns:
+                - arn:aws:iam::aws:policy/CloudWatchFullAccess
+                - arn:aws:iam::aws:policy/AWSXRayDaemonWriteAccess
+                - arn:aws:iam::aws:policy/AWSAppMeshEnvoyAccess
+#                - arn:aws:iam::aws:policy/AWSAppMeshPreviewEnvoyAccess
```

Comment:
Minor: commented policy ARN, can be removed.

Reply:
Done